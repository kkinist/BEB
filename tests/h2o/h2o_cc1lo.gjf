%chk=h2o.chk
%nprocs=4
%mem=1000mw
# CCSD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc

BEB step 8: low-spin CCSD(T) for cation of h2o

1 2

H O 0
cc-pVTZ
****

