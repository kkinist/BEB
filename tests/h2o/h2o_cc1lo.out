 Entering Gaussian System, Link 0=g09
 Initial command:
 /usr/local/apps/g09/l1.exe "/wrk/irikura/Gau-30258.inp" -scrdir="/wrk/irikura/"
 Entering Link 1 = /usr/local/apps/g09/l1.exe PID=     30260.
  
 Copyright (c) 1988,1990,1992,1993,1995,1998,2003,2009,2013,
            Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 09 program.  It is based on
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 09, Revision D.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, B. Mennucci, 
 G. A. Petersson, H. Nakatsuji, M. Caricato, X. Li, H. P. Hratchian, 
 A. F. Izmaylov, J. Bloino, G. Zheng, J. L. Sonnenberg, M. Hada, 
 M. Ehara, K. Toyota, R. Fukuda, J. Hasegawa, M. Ishida, T. Nakajima, 
 Y. Honda, O. Kitao, H. Nakai, T. Vreven, J. A. Montgomery, Jr., 
 J. E. Peralta, F. Ogliaro, M. Bearpark, J. J. Heyd, E. Brothers, 
 K. N. Kudin, V. N. Staroverov, T. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. Rendell, J. C. Burant, S. S. Iyengar, J. Tomasi, 
 M. Cossi, N. Rega, J. M. Millam, M. Klene, J. E. Knox, J. B. Cross, 
 V. Bakken, C. Adamo, J. Jaramillo, R. Gomperts, R. E. Stratmann, 
 O. Yazyev, A. J. Austin, R. Cammi, C. Pomelli, J. W. Ochterski, 
 R. L. Martin, K. Morokuma, V. G. Zakrzewski, G. A. Voth, 
 P. Salvador, J. J. Dannenberg, S. Dapprich, A. D. Daniels, 
 O. Farkas, J. B. Foresman, J. V. Ortiz, J. Cioslowski, 
 and D. J. Fox, Gaussian, Inc., Wallingford CT, 2013.
 
 ******************************************
 Gaussian 09:  ES64L-G09RevD.01 24-Apr-2013
                17-May-2017 
 ******************************************
 %chk=h2o.chk
 %nprocs=4
 Will use up to    4 processors via shared memory.
 %mem=1000mw
 ---------------------------------------------------
 # CCSD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc
 ---------------------------------------------------
 1/29=2,38=1/1;
 2/12=2,40=1/2;
 3/5=7,11=9,14=-4,16=1,24=10,25=1,30=1/1,2,3;
 4//1;
 5/5=2,8=3,13=1,38=5/2,8;
 8/6=4,9=120000,10=1/1,4;
 9/5=7,6=100,14=2/13;
 6/7=2,8=2,9=2,10=2/1;
 99/5=1,9=1/99;
 ----------------------------------------------
 BEB step 8: low-spin CCSD(T) for cation of h2o
 ----------------------------------------------
 Structure from the checkpoint file:  "h2o.chk"
 Charge =  1 Multiplicity = 2
 Redundant internal coordinates found in file.
 O,0,0.,0.,-0.1405046397
 H,0,0.,-0.7615598004,0.4580938899
 H,0,0.,0.7615598004,0.4580938899
 Recover connectivity data from disk.
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          8           0        0.000000    0.000000   -0.140505
      2          1           0        0.000000   -0.761560    0.458094
      3          1           0        0.000000    0.761560    0.458094
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3
     1  O    0.000000
     2  H    0.968656   0.000000
     3  H    0.968656   1.523120   0.000000
 Stoichiometry    H2O(1+,2)
 Framework group  C2V[C2(O),SGV(H2)]
 Deg. of freedom     2
 Full point group                 C2V     NOp   4
 Largest Abelian subgroup         C2V     NOp   4
 Largest concise Abelian subgroup C2      NOp   2
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          8           0        0.000000    0.000000    0.119720
      2          1           0        0.000000    0.761560   -0.478879
      3          1           0        0.000000   -0.761560   -0.478879
 ---------------------------------------------------------------------
 Rotational constants (GHZ):    787.9091958    432.3087586    279.1468894
 General basis read from cards:  (5D, 7F)
 AO basis set in the form of general basis input (Overlap normalization):
      1 0
 S   7 1.00       0.000000000000
      0.1533000000D+05  0.5198089434D-03
      0.2299000000D+04  0.4020256215D-02
      0.5224000000D+03  0.2071282673D-01
      0.1473000000D+03  0.8101055358D-01
      0.4755000000D+02  0.2359629851D+00
      0.1676000000D+02  0.4426534455D+00
      0.6207000000D+01  0.3570644227D+00
 S   6 1.00       0.000000000000
      0.5224000000D+03 -0.4421150084D-04
      0.1473000000D+03 -0.1225910413D-02
      0.4755000000D+02 -0.1055177248D-01
      0.1676000000D+02 -0.6744526326D-01
      0.6207000000D+01 -0.1711986073D+00
      0.6882000000D+00  0.1073298561D+01
 S   1 1.00       0.000000000000
      0.1752000000D+01  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.2384000000D+00  0.1000000000D+01
 P   3 1.00       0.000000000000
      0.3446000000D+02  0.4116348957D-01
      0.7749000000D+01  0.2577628359D+00
      0.2280000000D+01  0.8024192744D+00
 P   1 1.00       0.000000000000
      0.7156000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.2140000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.2314000000D+01  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.6450000000D+00  0.1000000000D+01
 F   1 1.00       0.000000000000
      0.1428000000D+01  0.1000000000D+01
 ****
      2 0
 S   3 1.00       0.000000000000
      0.3387000000D+02  0.2549486323D-01
      0.5095000000D+01  0.1903627659D+00
      0.1159000000D+01  0.8521620222D+00
 S   1 1.00       0.000000000000
      0.3258000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1407000000D+01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.3880000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1057000000D+01  0.1000000000D+01
 ****
      3 0
 S   3 1.00       0.000000000000
      0.3387000000D+02  0.2549486323D-01
      0.5095000000D+01  0.1903627659D+00
      0.1159000000D+01  0.8521620222D+00
 S   1 1.00       0.000000000000
      0.3258000000D+00  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.1027000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1407000000D+01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.3880000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1057000000D+01  0.1000000000D+01
 ****

 There are    27 symmetry adapted cartesian basis functions of A1  symmetry.
 There are     7 symmetry adapted cartesian basis functions of A2  symmetry.
 There are    12 symmetry adapted cartesian basis functions of B1  symmetry.
 There are    19 symmetry adapted cartesian basis functions of B2  symmetry.
 There are    23 symmetry adapted basis functions of A1  symmetry.
 There are     7 symmetry adapted basis functions of A2  symmetry.
 There are    11 symmetry adapted basis functions of B1  symmetry.
 There are    17 symmetry adapted basis functions of B2  symmetry.
    58 basis functions,    86 primitive gaussians,    65 cartesian basis functions
     5 alpha electrons        4 beta electrons
       nuclear repulsion energy         9.0882413160 Hartrees.
 NAtoms=    3 NActive=    3 NUniq=    2 SFac= 2.25D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=    58 RedAO= T EigKep=  3.67D-03  NBF=    23     7    11    17
 NBsUse=    58 1.00D-06 EigRej= -1.00D+00 NBFU=    23     7    11    17
 ExpMin= 1.03D-01 ExpMax= 1.53D+04 ExpMxC= 5.22D+02 IAcc=1 IRadAn=         1 AccDes= 0.00D+00
 Harris functional with IExCor=  205 and IRadAn=       1 diagonalized for initial guess.
 HarFok:  IExCor=  205 AccDes= 0.00D+00 IRadAn=         1 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=     500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Initial guess orbital symmetries:
 Alpha Orbitals:
       Occupied  (A1) (A1) (B2) (A1) (B1)
       Virtual   (A1) (B2) (B2) (A1) (A1) (B1) (B2) (A1) (A2) (B1)
                 (A1) (B2) (B2) (A1) (B1) (A2) (B2) (A1) (A1) (A1)
                 (B2) (B1) (A1) (B1) (A2) (B2) (B2) (A1) (A2) (B2)
                 (A1) (B1) (A2) (A1) (B1) (B2) (B2) (A1) (A1) (B2)
                 (B1) (A1) (B1) (A2) (A1) (B1) (B2) (A1) (A2) (A1)
                 (B2) (B2) (A1)
 Beta  Orbitals:
       Occupied  (A1) (A1) (B2) (A1)
       Virtual   (B1) (A1) (B2) (B2) (A1) (A1) (B1) (B2) (A1) (A2)
                 (B1) (A1) (B2) (B2) (A1) (B1) (A2) (B2) (A1) (A1)
                 (A1) (B2) (B1) (A1) (B1) (A2) (B2) (B2) (A1) (A2)
                 (B2) (A1) (B1) (A2) (A1) (B1) (B2) (B2) (A1) (A1)
                 (B2) (B1) (A1) (B1) (A2) (A1) (B1) (B2) (A1) (A2)
                 (A1) (B2) (B2) (A1)
 The electronic state of the initial guess is 2-B1.
 Initial guess <Sx>= 0.0000 <Sy>= 0.0000 <Sz>= 0.5000 <S**2>= 0.7500 S= 0.5000
 Keep R1 and R2 ints in memory in symmetry-blocked form, NReq=3449867.
 Requested convergence on RMS density matrix=1.00D-08 within  64 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(UHF) =  -75.6567585249     A.U. after   11 cycles
            NFock= 11  Conv=0.45D-08     -V/T= 1.9998
 <Sx>= 0.0000 <Sy>= 0.0000 <Sz>= 0.5000 <S**2>= 0.7571 S= 0.5035
 <L.S>= 0.000000000000E+00
 Annihilation of the first spin contaminant:
 S**2 before annihilation     0.7571,   after     0.7500
 QCSCF skips out because SCF is already converged.
 ExpMin= 1.03D-01 ExpMax= 1.53D+04 ExpMxC= 5.22D+02 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 HarFok:  IExCor=  205 AccDes= 0.00D+00 IRadAn=         5 IDoV=-2 UseB2=F ITyADJ=14
 ICtDFT= 12500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 Range of M.O.s used for correlation:     2    58
 NBasis=    58 NAE=     5 NBE=     4 NFC=     1 NFV=     0
 NROrb=     57 NOA=     4 NOB=     3 NVA=    53 NVB=    54
 Semi-Direct transformation.
 ModeAB=           4 MOrb=             4 LenV=    1048292615
 LASXX=        88542 LTotXX=       88542 LenRXX=      183223
 LTotAB=       94681 MaxLAS=      346788 LenRXY=           0
 NonZer=      271765 LenScr=      815104 LnRSAI=      346788
 LnScr1=     1039360 LExtra=           0 Total=      2384475
 MaxDsk=          -1 SrtSym=           T ITran=            4
 JobTyp=1 Pass  1:  I=   1 to   4.
 (rs|ai) integrals will be sorted in core.
 ModeAB=           4 MOrb=             3 LenV=    1048292615
 LASXX=        68438 LTotXX=       68438 LenRXX=      131786
 LTotAB=       63348 MaxLAS=      260091 LenRXY=           0
 NonZer=      200224 LenScr=      785920 LnRSAI=      260091
 LnScr1=      785920 LExtra=           0 Total=      1963717
 MaxDsk=          -1 SrtSym=           T ITran=            4
 JobTyp=2 Pass  1:  I=   1 to   3.
 (rs|ai) integrals will be sorted in core.
 Spin components of T(2) and E(2):
     alpha-alpha T2 =       0.6188555351D-02 E2=     -0.3073519152D-01
     alpha-beta  T2 =       0.3112136897D-01 E2=     -0.1529687566D+00
     beta-beta   T2 =       0.3024162851D-02 E2=     -0.1374766630D-01
 ANorm=    0.1019967689D+01
 E2 =    -0.1974516145D+00 EUMP2 =    -0.75854210139333D+02
 (S**2,0)=  0.75708D+00           (S**2,1)=  0.75115D+00
 E(PUHF)=      -0.75660354847D+02        E(PMP2)=      -0.75856304658D+02
 Keep R2 and R3 ints in memory in symmetry-blocked form, NReq=3411002.
 Iterations= 100 Convergence= 0.100D-06
 Iteration Nr.   1
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        34
 NAB=    12 NAA=     6 NBB=     3.
 E(PMP3)=      -0.75868864880D+02
 MP4(R+Q)=  0.28728054D-02
 E3=       -0.13516099D-01        EUMP3=      -0.75867726238D+02
 E4(DQ)=   -0.52045995D-03        UMP4(DQ)=   -0.75868246698D+02
 E4(SDQ)=  -0.13969477D-02        UMP4(SDQ)=  -0.75869123186D+02
 DE(Corr)= -0.20808700     E(Corr)=     -75.864845527    
 NORM(A)=   0.10229985D+01
 Iteration Nr.   2
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        34
 NAB=    12 NAA=     6 NBB=     3.
 DE(Corr)= -0.21164445     E(CORR)=     -75.868402977     Delta=-3.56D-03
 NORM(A)=   0.10245201D+01
 Iteration Nr.   3
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        34
 NAB=    12 NAA=     6 NBB=     3.
 DE(Corr)= -0.21253588     E(CORR)=     -75.869294401     Delta=-8.91D-04
 NORM(A)=   0.10248838D+01
 Iteration Nr.   4
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        34
 NAB=    12 NAA=     6 NBB=     3.
 DE(Corr)= -0.21284362     E(CORR)=     -75.869602141     Delta=-3.08D-04
 NORM(A)=   0.10249790D+01
 Iteration Nr.   5
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        34
 NAB=    12 NAA=     6 NBB=     3.
 DE(Corr)= -0.21288161     E(CORR)=     -75.869640131     Delta=-3.80D-05
 NORM(A)=   0.10249897D+01
 Iteration Nr.   6
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        34
 NAB=    12 NAA=     6 NBB=     3.
 DE(Corr)= -0.21288684     E(CORR)=     -75.869645361     Delta=-5.23D-06
 NORM(A)=   0.10249897D+01
 Iteration Nr.   7
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        34
 NAB=    12 NAA=     6 NBB=     3.
 DE(Corr)= -0.21288687     E(CORR)=     -75.869645399     Delta=-3.85D-08
 NORM(A)=   0.10249910D+01
 Iteration Nr.   8
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        34
 NAB=    12 NAA=     6 NBB=     3.
 DE(Corr)= -0.21288721     E(CORR)=     -75.869645731     Delta=-3.32D-07
 NORM(A)=   0.10249909D+01
 Iteration Nr.   9
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        34
 NAB=    12 NAA=     6 NBB=     3.
 DE(Corr)= -0.21288725     E(CORR)=     -75.869645778     Delta=-4.64D-08
 NORM(A)=   0.10249909D+01
 Iteration Nr.  10
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        34
 NAB=    12 NAA=     6 NBB=     3.
 DE(Corr)= -0.21288720     E(CORR)=     -75.869645722     Delta= 5.53D-08
 NORM(A)=   0.10249908D+01
 Largest amplitude= 2.59D-02
 Time for triples=        2.27 seconds.
 T4(CCSD)= -0.49030023D-02
 T5(CCSD)=  0.93096190D-04
 CCSD(T)= -0.75874455629D+02

 S**2, projected HF & approx projected MPn energies after annihilation of
 unwanted spin states (see manual for definitions):

 spins       (S**2,0) (S**2,1)    PUHF        PMP2        PMP3        PMP4
 annihilated
 s+1         0.74998  0.75001  -75.660355  -75.856305  -75.868865
 s+1,s+2     0.75000  0.75000  -75.660349  -75.856299  -75.868859
 s+1 to s+3  0.75000  0.75000  -75.660349  -75.856299  -75.868859

 Discarding MO integrals.

 **********************************************************************

            Population analysis using the SCF density.

 **********************************************************************

 Orbital symmetries:
 Alpha Orbitals:
       Occupied  (A1) (A1) (B2) (B1) (A1)
       Virtual   (A1) (B2) (B2) (A1) (A1) (B1) (A1) (B2) (A2) (B1)
                 (A1) (B2) (B2) (A1) (B1) (A2) (B2) (A1) (A1) (A1)
                 (B2) (B1) (A1) (A2) (B2) (B1) (B2) (A1) (A2) (B1)
                 (B2) (A1) (A2) (A1) (B1) (B2) (B2) (A1) (A1) (B2)
                 (B1) (A1) (B1) (A2) (B1) (A1) (B2) (A1) (A2) (A1)
                 (B2) (B2) (A1)
 Beta  Orbitals:
       Occupied  (A1) (A1) (B2) (A1)
       Virtual   (B1) (A1) (B2) (B2) (A1) (A1) (A1) (B2) (B1) (A2)
                 (B1) (A1) (B2) (B2) (A1) (B1) (A2) (B2) (A1) (A1)
                 (A1) (B2) (B1) (A1) (A2) (B2) (B1) (B2) (A1) (A2)
                 (B1) (B2) (A1) (A2) (A1) (B2) (B1) (B2) (A1) (A1)
                 (B2) (B1) (A1) (B1) (A2) (A1) (B1) (B2) (A1) (A2)
                 (A1) (B2) (B2) (A1)
 The electronic state is 2-B1.
 Alpha  occ. eigenvalues --  -21.13108  -1.90604  -1.20815  -1.13077  -1.09495
 Alpha virt. eigenvalues --   -0.15958  -0.07924   0.18922   0.23647   0.30667
 Alpha virt. eigenvalues --    0.34265   0.43745   0.45390   0.48334   0.57731
 Alpha virt. eigenvalues --    0.72531   0.85025   1.12979   1.14344   1.55900
 Alpha virt. eigenvalues --    1.57962   1.66918   1.73368   1.76432   2.16236
 Alpha virt. eigenvalues --    2.51911   2.91822   3.05469   3.15035   3.23766
 Alpha virt. eigenvalues --    3.27347   3.43769   3.48605   3.53579   3.58405
 Alpha virt. eigenvalues --    3.64344   3.73219   3.86076   3.92744   4.14412
 Alpha virt. eigenvalues --    4.14604   4.41031   4.63486   4.74193   4.99297
 Alpha virt. eigenvalues --    5.50920   6.02913   6.32468   6.34028   6.45493
 Alpha virt. eigenvalues --    6.48640   6.52376   6.63151   6.69995   6.92897
 Alpha virt. eigenvalues --    7.28778   7.75299  12.18298
  Beta  occ. eigenvalues --  -21.08770  -1.74787  -1.17079  -1.04444
  Beta virt. eigenvalues --   -0.31711  -0.14785  -0.07268   0.18757   0.25169
  Beta virt. eigenvalues --    0.31865   0.44303   0.46516   0.48753   0.50492
  Beta virt. eigenvalues --    0.59882   0.76226   0.84840   1.13751   1.17297
  Beta virt. eigenvalues --    1.62780   1.65059   1.67496   1.74365   1.83688
  Beta virt. eigenvalues --    2.16145   2.52833   2.92967   3.06402   3.15665
  Beta virt. eigenvalues --    3.23779   3.36498   3.43459   3.48290   3.53896
  Beta virt. eigenvalues --    3.61152   3.64986   3.74493   3.87261   3.93350
  Beta virt. eigenvalues --    4.15222   4.22003   4.41656   4.67346   4.78423
  Beta virt. eigenvalues --    5.04185   5.55372   6.06842   6.40254   6.44137
  Beta virt. eigenvalues --    6.50850   6.52304   6.54209   6.69443   6.73733
  Beta virt. eigenvalues --    6.97004   7.29938   7.76944  12.21672
          Condensed to atoms (all electrons):
               1          2          3
     1  O    7.066109   0.310305   0.310305
     2  H    0.310305   0.355583  -0.009248
     3  H    0.310305  -0.009248   0.355583
          Atomic-Atomic Spin Densities.
               1          2          3
     1  O    1.102569  -0.010350  -0.010350
     2  H   -0.010350  -0.031546   0.000961
     3  H   -0.010350   0.000961  -0.031546
 Mulliken charges and spin densities:
               1          2
     1  O    0.313281   1.081869
     2  H    0.343360  -0.040935
     3  H    0.343360  -0.040935
 Sum of Mulliken charges =   1.00000   1.00000
 Mulliken charges and spin densities with hydrogens summed into heavy atoms:
               1          2
     1  O    1.000000   1.000000
 Electronic spatial extent (au):  <R**2>=             14.2997
 Charge=              1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.0000    Y=              0.0000    Z=             -2.2670  Tot=              2.2670
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             -4.6903   YY=             -2.0262   ZZ=             -4.1918
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             -1.0542   YY=              1.6100   ZZ=             -0.5557
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              0.0000  YYY=              0.0000  ZZZ=             -1.4736  XYY=              0.0000
  XXY=              0.0000  XXZ=             -0.2213  XZZ=              0.0000  YZZ=              0.0000
  YYZ=             -1.9089  XYZ=              0.0000
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=             -2.6131 YYYY=             -2.6126 ZZZZ=             -4.1950 XXXY=              0.0000
 XXXZ=              0.0000 YYYX=              0.0000 YYYZ=              0.0000 ZZZX=              0.0000
 ZZZY=              0.0000 XXYY=             -1.2522 XXZZ=             -1.1882 YYZZ=             -0.6420
 XXYZ=              0.0000 YYXZ=              0.0000 ZZXY=              0.0000
 N-N= 9.088241315969D+00 E-N=-1.936421631613D+02  KE= 7.567394579530D+01
 Symmetry A1   KE= 6.888086037569D+01
 Symmetry A2   KE= 4.477690104536D-33
 Symmetry B1   KE= 2.705148766438D+00
 Symmetry B2   KE= 4.087936653176D+00
 1\1\GINC-GAMBA\SP\UCCSD(T)-FC\Gen\H2O1(1+,2)\IRIKURA\17-May-2017\0\\# 
 CCSD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc\\BEB step 8: low-spi
 n CCSD(T) for cation of h2o\\1,2\O,0,0.,0.,-0.1405046397\H,0,0.,-0.761
 5598004,0.4580938899\H,0,0.,0.7615598004,0.4580938899\\Version=ES64L-G
 09RevD.01\State=2-B1\HF=-75.6567585\MP2=-75.8542101\MP3=-75.8677262\MP
 4D=-75.8711195\MP4DQ=-75.8682467\PUHF=-75.6603548\PMP2-0=-75.8563047\P
 MP3-0=-75.8688649\MP4SDQ=-75.8691232\CCSD=-75.8696457\CCSD(T)=-75.8744
 556\S2=0.757078\S2-1=0.751152\S2A=0.750029\RMSD=4.518e-09\PG=C02V [C2(
 O1),SGV(H2)]\\@


 JUST WHEN YOU THINK YOU'VE GOT THE WORLD ON A STRING,
 YOU FIND OUT IT'S YOUR LEASH.
 Job cpu time:       0 days  0 hours  0 minutes 23.7 seconds.
 File lengths (MBytes):  RWF=     25 Int=      0 D2E=      0 Chk=      1 Scr=      1
 Normal termination of Gaussian 09 at Wed May 17 13:36:42 2017.
